src/com/jambit/DatabaseConnection.java

```diff
@@ -38,19 +59,27 @@ private void connect() throws SQLException, IOException {
     databaseProps.load(new FileInputStream(PROPERTY_PATH));
 
     try {
```

Comment:
add a comment why you need this line. E.g. "tomcat requires to check class existence. Otherwise tomcat crashes."